### PR TITLE
chore: dual support tunnistamo/keycloak in authorization field

### PR DESCRIPTION
**Pull request by voneiden**

### CODE REVIEW

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Review by danipran — APPROVED** [review]

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

File: kerrokantasi/settings/test_settings.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from . import *  # noqa
+
+# Using globals() to avoid issues with type checking which most likely
+# can not follow the use of globals().update in the settings.__init__
+
+OIDC_API_TOKEN_AUTH = globals()["OIDC_API_TOKEN_AUTH"]
+OIDC_API_TOKEN_AUTH["API_AUTHORIZATION_FIELD"] = "authorization.permissions.scopes"
+OIDC_API_TOKEN_AUTH["OIDC_API_ISSUER"] = "http://test.local:8000/openid"
```
Comment (voneiden) [inline kerrokantasi/settings/test_settings.py:8]: nämä listoiksi

**Review by tuomas777 — APPROVED** [review]
LGTM 👍

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**KERROKANTASI-API branch is deployed to platta: https://kerrokantasi-pr470.api.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**